pyproject.toml

```diff
@@ -55,7 +55,7 @@ atom2dens = "powerfit_em.scripts.atom2dens:main"
 where = ["src"]
 
 [tool.setuptools.package-data]
-powerfit_em = ["data/*.npy", "kernels.cl", "_powerfit.pyx"]
+powerfit_em = ["data/*.npy", "correlators/kernels.cl", "_powerfit.pyx"]
```

Comment:
Is it correct that `_extensions.c` isn't in this list? @sverhoeven

Reply:
it is not needed in binary wheels, it should be in the sdist